### ISSUE TITLE
[Request] [QoL] Selecting different gachas

### ISSUE THREAD

**Issue by nihaals** [lcy0x1/BC-Tracker#3]
When you select a different gacha in the bottom right, can it change the selection in the top right?

**Comment by lcy0x1** [comment]
when you say top, you mean the sr and ur rate?

**Comment by nihaals** [comment]
The top right which decides the list of ubers in that gacha

**Comment by lcy0x1** [comment]
you mean bottom left and top right?

**Comment by lcy0x1** [comment]
no. top right is for you to filter the uber. if you select uberfest, but you know it's wargod uber, you won't want to search it in the uber sea

**Comment by nihaals** [comment]
I don't mean for it to be merged into one, but just set your select for you, as if it is a specific gacha, then you have to select the same one in both. In the smaller amount of cases with uberfest, you would have to select your gacha which you normally have to do anyway.

**Comment by lcy0x1** [comment]
ok. That will be in my to-do list also.
